apps/mobile/shared/lib/bridge.ts

```
@@ -45,12 +45,21 @@ export const appBridge = bridge({
           throw new Error('백엔드 응답에 accessToken이 없습니다.');
         }
 
-        const { accessToken, refreshToken } = result.data;
+        const {
+          accessToken,
+          refreshToken,
+          isNewUser,
```

Biome (2.4.4)

[error] Shouldn't redeclare 'isNewUser'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)